Source: Vratislav Podzimek <vpodzime@redhat.com>
Upstream: unknown
Reason: Fixes compilation under musl libc.

--- lib/libvolume_key.h
+++ lib/libvolume_key.h
@@ -18,8 +18,6 @@ Author: Miloslav Trmaƒç <mitr@redhat.com> */
 #ifndef LIBVOLUME_KEY_H__
 #define LIBVOLUME_KEY_H__
 
-#include <config.h>
-
 #include <cert.h>
 #include <glib.h>
 
